.github/integration/tests/sda/99_auth_test.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+set -e
+
+python -m pip install --upgrade pip
+pip install tox
+
+tox -e unit_tests -c /tests/sda/auth/tox.ini
```

Comment:
Should add an Issue for adding integration testing for the OIDC case? and possibly refactor the EGA test as well..

Reply:
The entire sda test suite are getting a bit to big to have all in one, so an issue for cleaning up the tests is a good idea.